### ISSUE TITLE
[Bug]: Banner.jsx

### ISSUE THREAD

**Issue by parasss19** [priyankarpal/ProjectsHut#956]
### What happened?

I found a BUG 🐞
Hi @priyankarpal 
The  banner image is not align correctly it has some extra gap sidewise

### Add screenshots

Add screenshots

https://github.com/priyankarpal/ProjectsHut/assets/101205226/62442a51-38f6-4efa-8b8f-ea18093e5078

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by priyankarpal** [comment]
yeah! we need to fix this issue for a better developer experience.

assigned to you @parasss19

**Comment by parasss19** [comment]
> yeah! we need to fix this issue for a better developer experience. assigned to you @parasss19 are you a GSSoC contributor? please add the good issue title

Yes I'm GSSOC'23 Contributor 
I will keep in mind 👍
